.github/workflows/scheduled.yml

```
@@ -396,3 +396,72 @@ jobs:
           name: join-fuzzer-failure-artifacts
           path: |
             /tmp/join_fuzzer_repro
+
+
+  presto-java-aggregation-fuzzer-run:
+    runs-on: 8-core
+    container: ghcr.io/facebookincubator/velox-dev:presto-java
+    timeout-minutes: 120
+    if: "$${{ github.event_name != 'pull_request' }}"
+    env:
+      CCACHE_DIR: "${{ github.workspace }}/.ccache/"
+      CCACHE_BASEDIR: "${{ github.workspace }}"
+      LINUX_DISTRO: "centos"
+    steps:
+
+      - name: "Restore ccache"
+        uses: assignUser/stash/restore@v1
+        with:
+          path: "${{ env.CCACHE_DIR }}"
+          key: ccache-fuzzer
```

```suggestion
          key: ccache-presto-java-fuzzer
```
Otherwise this will overwrite  the ccache for ubuntu (and the ubuntu ccache is not applicable for this as it's build with a different compiler)

---

You will also need to add an explicit save step (see `compile:`) to actually save the stash.

---

cant use the same cache since this one is on a docker container I suppose..

---

Yeah the other one is ubuntu gcc 11  and this one is centos and gcc 9 so the objects are not compatible. Would only blow up the cache size with no gain